data/json/items/gun/223.json

```diff
@@ -45,6 +45,29 @@
     "min_cycle_recoil": 1350,
     "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ]
   },
+  {
+    "id": "famas",
+    "copy-from": "rifle_auto",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "FAMAS G2" },
+    "description": "An assault rifle that was until recently used by French armed forces.  Famous for its high rate of fire, it was never available in a civilian version.",
```

Comment:
You'll need to change the description here to represent it as one of the Century Imports.

Reply:
Done.